.eslintrc

```diff
@@ -1,6 +1,230 @@
+    "one-var": [1, "never"], // allow just one var statement per function (off by default)
+    "operator-assignment": [1, "always"], // require assignment operator shorthand where possible or prohibit it entirely (off by default)
+    "padded-blocks": [1, "never"], // enforce padding within blocks (off by default)
+    "quotes": ["error", "single", { "allowTemplateLiterals": true }], // specify whether double or single quotes should be used
+    "semi": [1, "never"], // require or disallow use of semicolons instead of ASI
+    "semi-spacing": [1, {"before": false, "after": true}], // enforce spacing before and after semicolons
+    "sort-vars": 0, // sort variables within the same declaration block (off by default)
+    "space-before-blocks": [1, "always"], // require or disallow space before blocks (off by default)
+    "space-before-function-paren": [0], // require or disallow space before function opening parenthesis (off by default)
+    "object-curly-spacing": [1, "always"], // require or disallow spaces inside brackets (off by default)
+    "space-in-parens": [1, "never"], // require or disallow spaces inside parentheses (off by default)
+    "space-infix-ops": ["error", {"int32Hint": false}], // require spaces around operators
+    "keyword-spacing": [1, {"before": true}], // require a space after return, throw, and case
+    "space-unary-ops": [1, {"words": true, "nonwords": false}], // Require or disallow spaces before/after unary operators (words on by default, nonwords off by default)
+    "spaced-comment": [1, "always"], // require or disallow a space immediately following the // in a line comment (off by default)
+    "wrap-regex": 0, // require regex literals to be wrapped in parentheses (off by default)
+
+    //
+    // ECMAScript 6
+    //
+    // These rules are only relevant to ES6 environments and are off by default.
+    //
+    "no-var": 2, // require let or const instead of var (off by default)
+    "generator-star-spacing": [2, "before"], // enforce the spacing around the * in generator functions (off by default)
+
+    //
+    // Legacy
+    //
+    // The following rules are included for compatibility with JSHint and JSLint.
+    // While the names of the rules may not match up with the JSHint/JSLint counterpart,
+    // the functionality is the same.
+    //
+    "max-depth": [0, 3], // specify the maximum depth that blocks can be nested (off by default)
+    "max-len": [2, 100, 2], // specify the maximum length of a line in your program (off by default)
+    "max-params": [2, 5], // limits the number of parameters that can be used in the function declaration. (off by default)
+    "max-statements": 0, // specify the maximum number of statement allowed in a function (off by default)
+    "no-bitwise": 0, // disallow use of bitwise operators (off by default)
+    "no-plusplus": 2, // disallow use of unary operators, ++ and -- (off by default)
```

Comment:
Where did you get those rules from?
For better readability I would recommend using the https://www.npmjs.com/package/eslint-config-airbnb-base ruleset from eslint. You can simple extend those rules https://eslint.org/docs/user-guide/configuring#extending-configuration-files and you don't have to define them separately.